src/Microsoft.Graph/Models/Generated/GiphyRatingType.cs

```diff
@@ -20,14 +20,14 @@ public enum GiphyRatingType
     {
     
         /// <summary>
-        /// Moderate
```

Comment:
Any idea how this happened?  @MIchaelMainer

Reply:
This was a change in the latest metadata update on Tuesday.
https://github.com/microsoftgraph/msgraph-metadata/commit/cf293dcc71b9336c294418ea609dc1b4e5d599a5